galcheat/data/README.md

```diff
@@ -0,0 +1,118 @@
+`galcheat` data files
+=====================
+
+The following document describes the parameters expected for the photometric surveys and their filters: how they should be computed, their units, etc.
+
+Survey parameters
+-----------------
+### Units and types
+
+| parameter name    | type  | units          |
+| ----------------- | ----- | -------------- |
+| name              | str   | –              |
+| pixel_scale       | float | arcsec / pixel |
+| gain              | float | e- / ADU       |
+| mirror_diameter   | float | m         |
+| obscuration       | float | dimensionless  |
+| zeropoint_airmass | float | dimensionless  |
+
+### Description
+
+#### `name`
+
+The classical name or abbreviation for the survey. Most often this is how the survey is referred to.
+In case of an ambiguity, for instance when a survey has several instruments, the name of the instrument should be added as a suffix (e.g. `Euclid_VIS`).
+
+#### `pixel_scale`
+
+Size of a square pixel on the sky.
+
+#### `gain`
+
+Conversion factor between the photo-electrons received by the camera and the digital counts after the amplification of the electronics.
+
+#### `mirror_diameter`
+
+Primary mirror diameter, in meters.
+
+#### `obscuration`
+
+Proportion of the total area of the telescope that is obscured by the position of secondary mirrors, lenses, camera, etc.
+
+This parameter is used to compute the effective area of the telescope.
+
+#### `zeropoint_airmass`
+
+Airmass value at which the zeropoint is computed.
+
+Filter parameters
+-----------------
+### Units and types
+
+| parameter name | type      | units          |
+| -------------- | --------- | -------------- |
+| name           | str       | –              |
+| sky_brightness | float     | mag / arcsec^2 |
+| exp_time       | int/float | s              |
+| psf_fwhm       | float     | arcsec         |
+| zeropoint      | float     | e- / s         |
+
+### Description
+#### `name`
+
+Name of the filter
+
+#### `sky_brightness`
+
+Average sky brightness computed for the survey and this filter.
+
+#### `exp_time`
+
+Average exposure time of the filter on the same spot in the sky over the course of the survey.
+
+#### `psf_fwhm`
+
+Average full width at half-maximum (FWHM) of the point spread function (PSF) over the filter.
+
+This parameter is traditionally computed with the [`speclite`][speclite] library.
```

Comment:
Hi @aboucaud, thanks for writing this readme ! I would just have one comment about the `psf_fwhm` description. I don't think we compute it with speclite, or did I miss something ?

Reply:
You're right, I got confused with the zeropoint.

Reply:
Yes this what I thought ! Other than that I think it is fine !